README.md

```diff
@@ -2,4 +2,101 @@
 
 **THIS IS DRAFT AF. DON'T READ IT IF YOU DON'T LIKE CONFUSION.**
 
-## TOC
\ No newline at end of file
+## TOC
+
+* [Create, Consume, and Provide Context](#create-consume-and-provide-context)
+
+## Create, Consume, and Provide Context
+
+Context is a 3-part system.
+You **create** it, **consume** it, and **provide** it.
+
+**Create, consume, provide**. Create, consume, provide... rinse and repeat.
+
+### Create Context
+
+Create a Context using React's `createContext` function.
+
+```jsx
+import React from "react";
+
+let NameameContext = React.createContext();
```

Comment:
typo

Reply:
thanks. don't worry too much about code typos. i'll be a good boy and run them all before posting. I promise 😅